cmd/operator-sdk/bundle/create.go

```diff
@@ -46,59 +46,76 @@ func newCreateCmd() *cobra.Command {
+be written if '--generate-only=true'. Note that, if the 'manifests/' do not exist, this command will 
+create the directory which is expected with the CRD's resources from the '/deploy/crds/' directory and generate 
+a CSV file that can be applied to the cluster using standard tooling like kubectl and defines its 
+integration with OLM. 
+
+Also, the /metadata directory will be created with resources that are used to store supporting 
+metadata associated with the operator.
+		
+Notes:
+* The image is not runnable.
+* More info on Operator Bundle images and metadata format see: 
+  https://github.com/operator-framework/operator-registry#manifest-format 
+`,
+		Example: `
+The following invocation will build a memcached-operator bundle metadata locally which is useful 
+if you want to build an operator's bundle image manually, modify metadata before building an image, 
+or want to generate a 'manifests/' directory containing your operator manifests for compatibility
+with other operator tooling.
 
-To write all files required to build a bundle image without building the
-image, set '--generate-only=true'. A bundle.Dockerfile and bundle metadata
-will be written if '--generate-only=true':
+  $ operator-sdk bundle create --generate-only
+		
+  # The following is the directory of an operator registry bundle which will be generated.
 
-` + "```" + `
-  $ operator-sdk bundle create --generate-only --directory ./deploy/olm-catalog/test-operator/manifests
   $ ls .
   ...
   bundle.Dockerfile
   ...
-  $ tree ./deploy/olm-catalog/test-operator/
-  ./deploy/olm-catalog/test-operator/
-  ├── manifests
-  │   ├── example.com_tests_crd.yaml
-  │   └── test-operator.clusterserviceversion.yaml
+  $ tree ./deploy/olm-catalog/memcached-operator/
+  └── manifests
+  	└── cache.example.com_memcacheds_crd.yaml
+  	└── memcached-operator.clusterserviceversion.yaml
   └── metadata
-      └── annotations.yaml
-` + "```" + `
-
-'--generate-only' is useful if you want to build an operator's bundle image
-manually or modify metadata before building an image.
+  	└── annotations.yaml
+		
+  # Now, you can, for example, validate the bundle generate locally with:
+  $ operator-sdk bundle validate \
+	--directory ./deploy/olm-catalog/memcached-operator/	
+		
+The following invocation will build a test-operator 0.1.0 bundle image using Docker. 
 
-More information on operator bundle images and the manifests/metadata format:
-https://github.com/openshift/enhancements/blob/master/enhancements/olm/operator-bundle.md
-
-NOTE: bundle images are not runnable.
-`,
-		Example: `The following invocation will build a test-operator 0.1.0 bundle image using Docker.
-This image will contain manifests for package channels 'stable' and 'beta':
+  # This image will contain manifests for package channels 'stable' and 'beta':
 
   $ operator-sdk bundle create quay.io/example/test-operator:v0.1.0 \
-      --directory ./deploy/olm-catalog/test-operator/manifests \
```

Comment:
The correct path is `./deploy/olm-catalog/test-operator` and not `./deploy/olm-catalog/test-operator/manifests`. It will NOT work with manifests/